trl/trainer/grpo_config.py

```diff
@@ -115,6 +115,10 @@ class GRPOConfig(TrainingArguments):
             applied. The [Dr. GRPO](https://github.com/sail-sg/understand-r1-zero/blob/main/understand-r1-zero.pdf)
             paper recommends not scaling the rewards, as scaling by the standard deviation introduces a question-level
             difficulty bias.
+        mask_truncated_samples (`bool`, *optional*, defaults to `False`):
```

Comment:
turbo nit: since we use "completions" elsewhere in the config, let's align it here too
```suggestion
        mask_truncated_completions (`bool`, *optional*, defaults to `False`):
```